Cleanup User Guide section on depletion (#2212)
openmc-dev/openmc: docs/source/usersguide/depletion.rst

.. note::

   Ideally, one-group cross section data should be available for every
   reaction in the depletion chain. If cross section data is not present for
   a nuclide in the depletion chain with at least one reaction, that reaction
   will not be simulated.

> Loading and Generating Microscopic Cross Sections
> -------------------------------------

Reviewer: ```suggestion
Loading and Generating Microscopic Cross Sections
-------------------------------------------------
```

As mentioned earlier, any transport code could be used to calculate one-group
microscopic cross sections. The :mod:`openmc.deplete` module provides the 
:class:`~openmc.deplete.MicroXS` class, which contains methods to read in

> pre-caluclated cross sections from a ``.csv`` file or from data arrays::

Reviewer: ```suggestion
pre-calculated cross sections from a ``.csv`` file or from data arrays::
```

   sure your cross sections are in the correct units before passing to a
   :class:`~openmc.deplete.IndependentOperator` object.

> If you are runnnig :meth:`~openmc.deplete.MicroXS.from_model()` on a cluster
> that does not share local filesystems across nodes, you'll need to set an

Reviewer: ```suggestion
If you are runnig :meth:`~openmc.deplete.MicroXS.from_model()` on a cluster
where temporary files are created on a local filesystem that is not shared across nodes, you'll need to set an
```

environment variable so that each MPI process knows where to store output files
used to calculate the microscopic cross sections. In order of priority, they

> are `TMPDIR`. `TEMP`, and `TMP`. Users interested in further details can read

Reviewer: ```suggestion
are :envvar:`TMPDIR`. :envvar:`TEMP`, and :envvar:`TMP`. Users interested in further details can read
```

> the `relevant docpage on the tempfile pacakge <https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir>`_

Reviewer: ```suggestion
the documentation for the `tempfile <https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir>`_ module.
```

Multiple Materials
~~~~~~~~~~~~~~~~~~

> A transport-independent depletion simulation using ``source-race`` normalization

Reviewer: ```suggestion
A transport-independent depletion simulation using ``source-rate`` normalization
```
